STYLE_GUIDELINES.md

```diff
@@ -174,7 +173,7 @@ arguments.
 
 ## New lines
 
--   Keep code under 80 columns. Break up statements if you must.
+-   Keep code under 80 columns where appropriate. Break up statements if you must.
```

Comment:
It seems that some of us are using 120 columns as the standard, and only splitting lines around the 80 mark if it improves readability.

Reply:
I'll make some tweaks to the wording. I too find myself regularly exceeding the 80 column limit but I feel it may still be worthwhile aiming for 80 provided it doesn't sacrifice readability